setup.cfg

```diff
@@ -1,2 +1,4 @@
 [metadata]
 description_file = README.md
+name = anovos
```

Comment:
The `name` is also defined in the `setup.py` file. Does one file take precedence over the other?